taginfo.json

```
@@ -1,6 +1,6 @@
 {
   "data_format": 1,
-  "data_url": "https://github.com/hiddewie/OpenRailwayMap-vector/blob/master/taginfo.json",
+  "data_url": "https://raw.githubusercontent.com/hiddewie/OpenRailwayMap-vector/blob/master/taginfo.json",
```

```suggestion
  "data_url": "https://raw.githubusercontent.com/hiddewie/OpenRailwayMap-vector/master/taginfo.json",
```

without the `blob` gives results for me